modules/musl-cross

```diff
@@ -32,22 +33,28 @@ musl-cross_libraries := \
 else
 
 # Force a full build of the cross compiler
-# have to build both x86_64 and i386 versions for coreboot
 
+ifeq "$(CONFIG_MUSL_TARGET)" "x86-64"
+# have to build both x86_64 and i386 versions for coreboot
 musl-cross_configure := \
-	/bin/echo -e >> Makefile 'both:' ; \
```

Comment:
@SergiiDmytruk the disappearance of this statement seems to be the cause of the failing in CI around line 1833 https://app.circleci.com/pipelines/github/tlaurion/heads/734/workflows/978d2ee9-07b5-4014-b570-b9f69a787d53/jobs/1106

Reply:
@tlaurion You mean this error?
```
cc1: error: no include path in which to search for stdc-predef.h
```
Not sure why renaming a target would cause this, but maybe something else does.

I haven't tested x86 build with my changes, that's a good point. For now I'm trying to solve issues with `/init` not being execute by the kernel (`error -2`). I'll fix x86 build once I get it to work on power to not fix it more than once.

Reply:
No, I was referring to the difference in output from last successful full build (not from cache) where weirdly
> cc1: error: no include path in which to search for stdc-predef.h

Is also shown, but where [it is not a problem](https://circleci.com/api/v1.1/project/github/tlaurion/heads/1058/output/103/0?file=true&allocation-id=60d24e2c9f1dbc44db03306d-0-build%2F2B2E18CF) for both i386 and x64
```

echo | /root/project/build/musl-cross-38e52db8358c043ae82b346a2e6e66bc86a53bc1/build/local/x86_64-linux-musl/obj_gcc/./gcc/xgcc -B/root/project/build/musl-cross-38e52db8358c043ae82b346a2e6e66bc86a53bc1/build/local/x86_64-linux-musl/obj_gcc/./gcc/ -E -dM - | \
  sed -n -e 's/^#define \([^_][a-zA-Z0-9_]*\).*/\1/p' \
	 -e 's/^#define \(_[^_A-Z][a-zA-Z0-9_]*\).*/\1/p' | \
  sort -u > tmp-macro_list
cc1: error: no include path in which to search for stdc-predef.h
/bin/bash ../../src_gcc/gcc/../move-if-change tmp-macro_list macro_list
echo timestamp > s-macro_list
```

```
echo | /root/project/build/musl-cross-38e52db8358c043ae82b346a2e6e66bc86a53bc1/build/local/x86_64-linux-musl/obj_gcc/./gcc/xgcc -B/root/project/build/musl-cross-38e52db8358c043ae82b346a2e6e66bc86a53bc1/build/local/x86_64-linux-musl/obj_gcc/./gcc/ -E -dM - | \
  sed -n -e 's/^#define \([^_][a-zA-Z0-9_]*\).*/\1/p' \
	 -e 's/^#define \(_[^_A-Z][a-zA-Z0-9_]*\).*/\1/p' | \
  sort -u > tmp-macro_list
cc1: error: no include path in which to search for stdc-predef.h
/bin/bash ../../src_gcc/gcc/../move-if-change tmp-macro_list macro_list
echo timestamp > s-macro_list
```

But where the path searched for compiled musl-cross is not found [here](https://circleci.com/api/v1.1/project/github/tlaurion/heads/1106/output/103/0?file=true&allocation-id=60e2325708b5c9541ae6c52d-0-build%2F507C5687):
```
../scripts/gcc-version.sh: line 26: /root/project/crossgcc/bin/x86-64-linux-musl-gcc: No such file or directory
../scripts/gcc-version.sh: line 27: /root/project/crossgcc/bin/x86-64-linux-musl-gcc: No such file or directory
```
Where that doesn't occur in x86 builds. will have to dig.

Reply:
Right, thanks for pointing it out. `x86-64` should be `x86_64` (`-` -> `_`).